openslides/assignments/static/js/assignments/pdf.js

```diff
@@ -367,58 +368,113 @@ angular.module('OpenSlidesApp.assignments.pdf', ['OpenSlidesApp.core.pdf'])
                 };
             };
 
-            var createTableBody = function(numberOfRows, sheetend) {
+            var createTableBody = function(numberOfRows, sheetend, maxballots) {
+                var ballotstoprint = numberOfRows * 2;
+                if (Number.isInteger(maxballots) && maxballots > 0 && maxballots < ballotstoprint) {
+                    ballotstoprint = maxballots;
+                }
                 var tableBody = [];
-                for (var i = 0; i < numberOfRows; i++) {
+                while (ballotstoprint > 1){
                     tableBody.push([createSection(sheetend), createSection(sheetend)]);
+                    ballotstoprint -= 2;
+                }
+                if (ballotstoprint == 1) {
+                    tableBody.push([createSection(sheetend), '']);
                 }
                 return tableBody;
             };
 
             var createContentTable = function() {
-
-                var tableBody = [];
+                // first, determine how many ballots we need
+                var amount;
+                var amount_method = Config.get('assignments_pdf_ballot_papers_selection').value;
+                switch (amount_method) {
+                    case 'NUMBER_OF_ALL_PARTICIPANTS':
+                        //TODO: check if getAll really includes all users
+                        amount = User.getAll().length;
+                        break;
+                    case 'NUMBER_OF_DELEGATES':
+                        //TODO: See above, same problem, but now we need all that are in a group
```

Comment:
You can filter the users with js-data (not tested):
`User.filter({where: { groups_id: 2}}).length`

see e.g. in  [agenda/projector.js](https://github.com/OpenSlides/OpenSlides/blob/ac60fb53daf8cc51a49cd3d834ef9f038779987f/openslides/agenda/static/js/agenda/projector.js#L107)